MAUI/TreeView/selection.md

```diff
@@ -0,0 +1,167 @@
+
+* `None`: Allows disabling the selection.
+* `Single`: Allows selecting the single item only. When clicking on the selected item, selection not will not be cleared. This is the default value for `SelectionMode`.
+* `SingleDeselect`: Allows selecting the single item only. When clicking on the selected item, selection gets cleared.
+* `Multiple`: Allows selecting more than one item. Selection is not cleared when selecting more than one items. When clicking on the selected item, selection gets cleared.
+* `Extended`: Allows to select the multiple items using the common key modifiers.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfTreeView x:Name="TreeView" SelectionMode="Multiple"/>
+{% endhighlight %}
+{% highlight c# %}
+TreeView.SelectionMode = SelectionMode.Multiple;
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI TreeView with Selection](Images/selection/MultipleSelection.png)
+
+## Programmatic Selection
+
+When the `SelectionMode` is other than `None`, the item or items in the TreeView can be selected from the code by setting the `SelectedItem`, or adding items to the `SelectedItems` property based on the `SelectionMode`.
+
+When the selection mode is `Single` or `SingleDeselect`, programmatically select an item by setting the underlying object to the `SelectedItem` property.
+
+{% tabs %}
+{% highlight c# %}
+
+treeView.SelectedItem = viewModel.CountriesInfo[2];
+
+{% endhighlight %}
+{% endtabs %}
+
+When the selection mode is `Multiple`, programmatically select more than one item by adding the underlying object to the `SelectedItems` property.
+
+{% tabs %}
+{% highlight c# %}
+
+treeView.SelectedItems.Add(viewModel.CountriesInfo[2]);
+treeView.SelectedItems.Add(viewModel.CountriesInfo[3]);
+
+{% endhighlight %}
+{% endtabs %}
+
+W> If an item is selected programmatically when `SelectionMode` is `None` and if multiple items are programmatically selected when `SelectionMode` is `Single` or `SingleDeselect`, then exception will be thrown internally.
+
+## Selected items 
+
+### Gets selected Items
+The TreeView gets all the selected items through the `SelectedItems` property and gets the single item by using the `SelectedItem` property.
+
+### Clear selected items
+The selected items can be cleared by calling the `SelectedItems.Clear()` method.
+
+{% tabs %}
+{% highlight c# %}
+treeView.SelectedItems.Clear();
+{% endhighlight %}
+{% endtabs %}
+ 
+### CurrentItem vs SelectedItem
+
+The TreeView gets the selected item by using the `SelectedItem` properties. Both `SelectedItem` and `CurrentItem` returns the same data object when selecting single item. When selecting more than one item, the `SelectedItem` property returns the first selected item, and the `CurrentItem` property returns the last selected item.

```

Comment:
The TreeView gets the selected item by using the `SelectedItem` properties. Both `SelectedItem` and `CurrentItem` return the same data object when selecting a single item. When selecting more than one item, the `SelectedItem` property returns the first selected item, and the `CurrentItem` property returns the last selected item.